ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/util/PersistentQueryCleanupImpl.java

```diff
@@ -59,6 +59,7 @@ public void cleanupLeakedQueries(final List<PersistentQueryMetadata> persistentQ
           new QueryCleanupService.QueryCleanupTask(
           serviceContext,
           appId,
+          "",
```

Comment:
Not quite sure what "leaked" queries are, but should we also check this for having shared runtimes enabled and potentially pass in the actual query id? @agavra can you explain the context here a bit?

Reply:
initially I introduced the QueryCleanupService to cleanup consumer groups from terminated queries (see https://github.com/confluentinc/ksql/pull/6160) asynchronously from the main clean up. It looks like since then it was also augmented to delete directories on startup that weren't properly deleted (due to some race conditions? If I recall this can happen if streams takes too long to close). 

I'm not familiar with how the shared runtimes work - if they share a state store then it makes sense not to run the query cleanup in that scenario.

I'm not entirely sure of this part though, looks like @lct45 and @vvcephei were involved in those changes, so if you're not confident we should pull them in.